pkg/build/apis/build/v1/conversion.go

```diff
@@ -19,7 +19,7 @@ func Convert_v1_BuildConfig_To_build_BuildConfig(in *v1.BuildConfig, out *newer.
 	// Strip off any default imagechange triggers where the buildconfig's
 	// "from" is not an ImageStreamTag, because those triggers
 	// will never be invoked.
-	imageRef := buildapihelpers.GetInputReference(out.Spec.Strategy)
+	imageRef := buildapihelpers.GetInputReferenceInternal(out.Spec.Strategy)
```

Comment:
can you use https://github.com/openshift/origin/blob/master/pkg/build/apis/build/internal_helpers/internal_helpers.go#L73 ?

`buildinternalhelpers.GetInputReference()`

(same for others)